src/components/VideoDetails/VideoDetails.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from './VideoDetails.module.scss';
+
+import CollapsibleText from '#src/components/CollapsibleText/CollapsibleText';
```

Comment:
Is there any reason why we always repeat the folder's name in the component's file? I mean why not to use `import CollapsibleText from '#src/components/CollapsibleText';` in the feature with the index.tsx file inside the folder.

Reply:
I don't suggest to change everything now :) We can do refactoring later.

Reply:
Hehe, that would mean changing every file indeed 😅  But it is a valid question. In our boilerplate project, we add an index.ts file for every component folder which re-exports the component. This is even done automatically via the generate component/container/screen command.

We can do that for sure, I don't really have an opinion on this.

Reply:
Using an index file can sometimes make it hard to navigate since you end up with 100 files in your project named index.  If it's just boilerplate that re-exports the main component that's fine, but it's another thing to maintain, so I would recommend we setup something like [plop](https://plopjs.com/) to help us if we want to go that route.